nilearn/group_sparse_covariance.py

```diff
@@ -545,7 +545,7 @@ def fit(self, subjects, y=None):
             _group_sparse_covariance, memory_level=1)(
                 self.covariances_, n_samples, self.alpha,
                 tol=self.tol, max_iter=self.max_iter,
-                verbose=self.verbose - 1, debug=False)
+                verbose=max(0, self.verbose - 1), debug=False)
```

Comment:
Not sure what's the best way to do it but @bcipolli could you remove the commits that are already in the #366 PR?

I believe you'll need to do that eventually when we want to merge into master.